### PR TITLE
Effort to put Course Info on TLT Systems

### PR DESCRIPTION
**Pull request by condaatje**
@arthurian Alright, this is pretty slimmed down to the things that TLT would be interested in. Tell me what you think, and if we can bring them in on it.

The commit messages have specifics about the changes that I'd like to draw attention to

### CODE REVIEW

File: dce_course_info/settings/aws.py [hunk 1]
```diff
@@ -12,12 +12,12 @@
```
Comment (condaatje) [inline dce_course_info/settings/aws.py:12]: -- variable names are lowercase
-- extraneous import statements commented out
-- Admin tuple was actually unused, has been commented out

File: dce_course_info/requirements/aws.txt [hunk 2]
```diff
@@ -14,4 +14,4 @@ djrill==1.3.0
 #django-postgrespool==0.3.0
```
Comment (condaatje) [inline dce_course_info/requirements/aws.txt:14]: -- Managed to get rid of .env file and its surrounding requirements
-- postgrespool and database-url were unused after using TLT database configuration settings
-- gunicorn is not going to be used in TLT AWS

File: gunicorn.py [hunk 3]
```diff
@@ -3,9 +3,9 @@
 # dotenv.read_dotenv()
```
Comment (condaatje) [inline gunicorn.py:3]: -- This was the last file using the .env file and dotenv/getenv reqs.

**Comment by arthurian** [comment]
@condaatje :+1: Much better! Let's bring in TLT and let them look it over.

**Comment by arthurian** [comment]
@bermudezjd @cmurtaugh @ekiczek Can you review the app and let us know if there's anything else that needs to be done to make it deployable?

**Comment by bermudezjd** [comment]
@arthurian @condaatje I think the best way for us to review this and the syllabus export would be to use github's `blame` feature and make any necessary comments in the settings files inline.  That way we can catch anything that may have been already merged between the projects.  So, I'd say go ahead and merge this in and we'll make any inline comments as needed on the set of commits.

**Comment by condaatje** [comment]
@bermudezjd @arthurian
Alright, sounds good.

**Pull request merged**